regress/unittests/win32compat/miscellaneous_tests.c

```diff
@@ -0,0 +1,176 @@
+#include "includes.h"
+#include <sys/ioctl.h>
+#include <sys/stat.h>
+#include <misc_internal.h>
+#include <sys/statvfs.h>
+#include <unistd.h>
+
+#include "../test_helper/test_helper.h"
+#include "tests.h"
+
+int retValue;
+
+// The ioctl() testcase is failing when ran from Run-OpenSSHUnitTest.
+//void 
+//test_ioctl()
+//{
+//	TEST_START("ioctl");
+//
+//	struct winsize ws;
+//	memset(&ws, 0, sizeof(ws));
+//	retValue = ioctl(fileno(stdin), TIOCGWINSZ, &ws);
```

Comment:
this test case is only valid is stdin is a tty. so you could put in the following condition
if (isatty(stdin))